### ISSUE TITLE
docker-index cve shall have json output

### ISSUE THREAD

**Issue by erickellerek1** [docker/index-cli-plugin#6]
The current tool evolved a bit in the last weeks, yet it becomes tricky to further automate with the current colorful output.
Basically there shall be a computerized format e.g. json where one could reuse the output for further processing.
Colors are nice when calling this tool as a user, not so nice to sed your way around when it comes to automation.

Any thoughts?

**Comment by cdupuis** [comment]
Yeah, that's good point. Was kind of expecting this to become an issue.
